core/spec/models/spree/promotion_spec.rb

```diff
@@ -384,6 +384,86 @@
     end
   end
 
+  describe '#not_started?' do
+    let(:promotion) { Spree::Promotion.new(starts_at: starts_at) }
+    subject { promotion.not_started? }
+
+    context 'no starts_at date' do
+      let(:starts_at) { nil }
+      it{ is_expected.to be_falsey }
```

Comment:
Could you add the missing space between `it` and the opening curly brace? :smile_cat: